src/test/harnesses/EigenHarness.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.27;
 import "../../contracts/token/Eigen.sol";
 
 contract EigenHarness is Eigen {
-    constructor(IERC20 _bEIGEN) Eigen(_bEIGEN) {}
+    constructor(IERC20 _bEIGEN) Eigen(_bEIGEN, "1.6.0") {}
```

Comment:
Hmm, I'm noticing that some of our harnesses wrap the underlying constructor by providing `version` as a constrcutor param, while others take the approach you have here - fill in a version string automatically.

I'm not sure this is a good PR to unify both of those styles (good followup work). However, I do think that we should be consistent with the style you're using here --

`DelegationManagerHarness` supplies `v9.9.9`, which intuitively makes sense to me as a "test version". Let's do that here.

(also there's the issue of whether to prepend with "v" - reviewing this, I noticed we're kinda doing this wrong in our release scripts. See Slack comment here: https://eigenlabs.slack.com/archives/C05PX94L3H9/p1747408010259309)

Reply:
(following discussion on slack, we should _not_ prepend with "v" as zeus currently does not supply versions with a leading "v")

Reply:
extracted the test version out as constant for harness classes